TestCases/cont_adj_incomp_navierstokes/cylinder/lam_incomp_cylinder_disc.cfg

```diff
@@ -33,16 +33,25 @@ WRT_BINARY_RESTART= NO
 % Read binary restart files (YES, NO)
 READ_BINARY_RESTART= NO
 
-% -------------------- INCOMPRESSIBLE FREE-STREAM DEFINITION ------------------%
+% ---------------- INCOMPRESSIBLE FLOW CONDITION DEFINITION -------------------%
 %
-% Free-stream density (1.2886 Kg/m^3 (air), 998.2 Kg/m^3 (water))
-FREESTREAM_DENSITY= 998.2
+% Initial density for incompressible flows (1.2886 kg/m^3 by default)
```

Comment:
I think the note somewhere in this file that the settings are for water would be nice - in the header if not here.